### ISSUE TITLE
Clarify: What's the difference to pyROOT?

### ISSUE THREAD

**Issue by klieret** [hsf-training/hsf-training-scikit-hep-webpage#7]
We got a lot of questions along the lines of "What's the difference between pyROOT and uproot", or "Why uproot if we have pyROOT" etc. 

Currently we don't mention pyROOT explicitly, so maybe this would make a nice callout box

**Comment by jpivarski** [comment]
I'm not sure if it's still true, but it used to be that this tutorial would come immediately after several PyROOT tutorials, so the distinction would come from context. But if this is a stand-alone thing, then having that would be useful.

This could help:

[![image](https://user-images.githubusercontent.com/1852447/179256574-040322ac-c99b-4c60-9605-c937c0185027.png)](https://github.com/jpivarski-talks/2021-09-13-star-uproot-awkward-tutorial/blob/main/fig/abstraction-layers.svg)

**Comment by jpivarski** [comment]
Yes, that's a good break down, but avoid the word "like" (emotional preference). Maybe "when you need to use..."? Also, "when" is less pejorative than "if".

**Comment by klieret** [comment]
- [ ] Also include link to the [pyROOT training](https://github.com/root-project/software-carpentry) if not already present
